package.json

```diff
@@ -26,11 +26,12 @@
     "request": "^2.69.0"
   },
   "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
Good point. I confused the use of `should.js` with the 'should' style of Chai assertions. Fixed in commit 3d145f3.